backend/inspirehep/records/marshmallow/utils.py

```diff
@@ -15,14 +15,15 @@ def get_display_name_for_author_name(author_name):
 
 
 def get_facet_author_name_for_author(author):
-    author_ids = author.get("ids", [])
-    author_bai = get_values_for_schema(author_ids, "INSPIRE BAI")
-    bai = author_bai[0] if author_bai else "BAI"
+    control_number = (
+        author["control_number"] if author.get("control_number") else "NOREC"
+    )
     author_preferred_name = get_value(author, "name.preferred_name")
 
     if author_preferred_name:
-        return "{}_{}".format(bai, author_preferred_name)
+        return "{}_{}".format(control_number, author_preferred_name)
```

Comment:
Use f-strings instead of `format` (wherever it makes sense):
```suggestion
        return f"{control_number}_{author_preferred_name}"
```